src/highlight-component.js

```diff
@@ -0,0 +1,81 @@
+function HighlightComponent() {
```

Comment:
JS should probably use strict mode, add `"use strict"` to the top to enable: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode (same with the test)

Reply:
fixed